honeybee_radiance_command/gensky.py

```diff
@@ -0,0 +1,221 @@
+"""gensky command."""
+
+from .options.gensky import GenskyOptions
+from ._command import Command
+import honeybee_radiance_command._exception as exceptions
+
+
+class Gensky(Command):
+    """gensky command."""
+
+    __slots__ = ('_ang', '_month', '_day', '_time', '_time_zone', '_solar_time',
+                 '_input')
+
+    def __init__(self, ang=None, month=None, day=None, time=None, time_zone=None,
+                 solar_time=False, options=None, output=None):
+        """Command.
+
+        Args:
+            ang: A tuple of altitude and azimuth values.
```

Comment:
`from_ang` should be implemented as a `classmethod` instead of mixing them with the current constructor.